unit/src/KDTreeTest.C

```diff
@@ -12,6 +12,15 @@
 
 #define TOL 1e-10
 
+// Make newer nanoflann API compatible with older nanoflann versions
+#if NANOFLANN_VERSION < 0x150
+namespace nanoflann
+{
+template <typename T, typename U>
+using ResultItem = std::pair<T, U>;
+}
+#endif
```

Comment:
Do we need this since it's in `KDTree.h` included here?

Reply:
We shouldn't!  Good catch.